app/containers/SearchContainer/index.js

```diff
@@ -0,0 +1,133 @@
+/**
+ *
+ * SearchContainer
+ *
+ */
+
+import React, { memo, useEffect } from 'react';
```

Comment:
as a convention for imports what we follow is 

// third party
// aliases
// relative

Reply:
please take a look at this one: https://drive.google.com/file/d/1R80yyKsIQlO7xrWTT0lIbP3jZs93LIxN/view?usp=sharing